### ISSUE TITLE
Tests for the monthly upsell

### ISSUE THREAD

**Issue by ScottDowne** [mozilla/donate.mozilla.org#1898]
I consider this a priority, but we're landing the upsell stuff without a huge focus on tests.

I'll do this after.

It needs some sort of tests.

It has some helper functions that can be tested easily, and also we can do some Selenium tests here too for an integration test.

**Comment by ScottDowne** [comment]
cc @gideonthomas

**Comment by ScottDowne** [comment]
@gideonthomas is this in the right milestone?

**Comment by gideonthomas** [comment]
yeah. We can punt it to the next one if we don't get it done on time
